app/controllers/stories.controller.js

```diff
@@ -2,17 +2,50 @@ const { where } = require("sequelize");
 const db = require("../models");
 const Story = db.story;
 const Op = db.Sequelize.Op;
-const { startCohereChat } = require('../services/cohere-client-service');
+const { startCohereChat, extendStory } = require('../services/cohere-client-service');
 
+/**
+ * Create new story
+ * @param {*} req 
+ * @param {*} res 
+ */
 exports.create = async (req, res) => {
-  try {
-    const userId = req.params.userId;
-    const storyParams = req.body.storyParams;
+  const userId = req.params.userId;
+  const storyParams = req.body.storyParams;
 
+  try {
     // Call the LLM service to generate the story text
-    const storyText = await startCohereChat(storyParams);
+    const preamble = 'You are a writer of children\'s stories';
+    let parametersText = '';
+
+    let chatInputMessage = 'Please write me a 5 paragraph childrens bedtime story. ';
+
+    if (storyParams.length !== 0) {
+      storyParams.forEach(parameter => {
+        parametersText += `${parameter.key}: ${parameter.value}, `;
+      });
+    }
+
+    if (parametersText !== '') {
+      chatInputMessage += ` Please use the following parameters to help build the content of the story. ${parametersText}`;
+    }
+
+    chatInputMessage += ` End the story extension in a way that leaves the plot open for continuation. 
+    Do not say explicitaly that there will be another chapter, but rather build the plot in a 
+    way that allows space for more story to be generated. Do not use phrases like "The End...for now." OR "To be continued..."`;
```

Comment:
some very explicit instructions here, did you find out by testing that such things were necessary to prevent Cohere's LLM from generating exactly such things?

Reply:
Yes, anytime I would suggest that a plot can be continued, the LLM had the tendency to add something like "The End...for now." OR "To be continued..." at the end of the story. I wanted to lean toward the story being stand alone, but flexible enough to extend the plot in a later story without explicitly placing words that would indicate it.